tensorflow/python/lib/io/file_io.py

```diff
@@ -819,13 +819,18 @@ def _make_full_path(parent, item):
     # Since `os.path.join` discards paths before one that starts with the path
     # separator (https://docs.python.org/3/library/os.path.html#os.path.join),
     # we have to manually handle that case as `/` is a valid character on GCS.
+    print("_make_full_path -> item = ", item)
```

Comment:
Please revert the debug prints

Reply:
Should be done. Thank you for calling it out before I forgot.